README.CUDA

```diff
@@ -0,0 +1,40 @@
+Support for NVIDIA GPUs via CUDA is currently experimental and many
```

Comment:
Yes, I'll just remove it. Maybe worth adding a note about `-DENABLE_CUDA` to the "options and features" section of `INSTALL` though?